.envrc

```diff
@@ -52,10 +52,11 @@ export OKTA_TEST_PASSWORD=
 export OKTA_TEST_SECRET=
 
 # Launch Darkly account
-export LD_SDK_KEY=
+export LD_SDK_KEY=sdk-1d0022eb-abe4-4b46-b0c3-162c69f55d45
```

Comment:
If you need these set, you should do them in the non-checked-in `.envrc.local`

Reply:
Yep, I've reset the SDK key, but @Igarfinkle can you delete that old key from the .envrc? I think the blank `export LD_SDK_KEY=` was used a placeholder to indicate that devs would need to add that variable to their .envrc.local.